ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -68,35 +84,35 @@ Write-Host "Locating '$imageName' image for use in template virtual machine"
 $imageObject = $labPlan | Get-AzLabServicesPlanImage | Where-Object {$_.DisplayName -EQ $imageName -and $_.Sku -EQ $sku -and (-not [string]::IsNullOrEmpty($_.EnabledState))} | Where-Object -Property EnabledState -eq "Enabled"
 
 if($null -eq $imageObject) {
-    Write-Error "Image '$imageName' was not found in the gallery images. No lab was created within lab account $labPlanName."
+    Write-Error "Image '$imageName' was not found in the gallery images. Couldn't create lab $labName."
     exit -1
 }
 
-# Create lab on the lab account
+# Create lab using the lab plan
 Write-Host "Creating $labName with '$($imageObject.Name)' image"
 Write-Warning "  Warning: Creating template vm may take up to 20 minutes."
 $lab = New-AzLabServicesLab -Name $labName `
         -ResourceGroupName $rgName `
         -Location $Location `
         -LabPlanId $labPlan.Id.ToString() `
         -AdditionalCapabilityInstallGpuDriver Disabled `
-        -AdminUserPassword (ConvertTo-SecureString $password -AsPlainText -Force) `
-        -AdminUserUsername "adminUser" `
+        -AdminUserPassword $(ConvertTo-SecureString $Password -AsPlainText -Force) `
+        -AdminUserUsername $UserName `
         -AutoShutdownProfileShutdownOnDisconnect Disabled `
```

Comment:
Should we have the idle settings off by default?  I think it might be better if they are on as a best practice?

Reply:
I didn't even notice that.  Updated the code to enable idle shutdown.